### PR TITLE
Update Apache Tika

### PR DESCRIPTION
**Pull request by evandongen**
upgrade Tika dependency

### CODE REVIEW

File: aspose/pom.xml [hunk 1]
```diff
@@ -27,197 +27,26 @@
...
-		</dependency>
 
 		<!-- Tika library -->
 		<dependency>
 			<groupId>org.apache.tika</groupId>
-			<artifactId>tika-parsers</artifactId>
-			<version>1.28.5</version>
+			<artifactId>tika-parser-microsoft-module</artifactId>
+			<version>2.9.2</version>
+			<type>pom</type>
```
Comment (nielsm5) [inline aspose/pom.xml:36]: type pom?

Comment (erikvandongen) [reply aspose/pom.xml:36]: fixed

**Review by nielsm5 — APPROVED** [review]

File: core/pom.xml [hunk 2]
```diff
@@ -646,7 +646,7 @@
 		<dependency>
 			<groupId>org.apache.tika</groupId>
 			<artifactId>tika-core</artifactId>
-			<version>1.28.5</version>
+			<version>2.9.2</version>
 			<exclusions>
 				<exclusion><!-- excluded because we want to ignore this (1.2.2) version! this dependency handles javax.activation.* imports, and is NOT compatible with the IBM JRE -->
```
Comment (jkosternl) [inline core/pom.xml:651]: Are you sure this exclusion is still needed? In the above `aspose/pom.xml`, you've removed it.

**Review by jkosternl — APPROVED** [review]

File: core/pom.xml (hunk 2 above)
Comment (jkosternl) [reply core/pom.xml:651]: Yes, but I was just thinking that this library isn't a dependency of the tika-core anymore.

Comment (evandongen) [reply core/pom.xml:651]: Good point, I'll remove the comment and exclusion

**Pull request merged**